proco/connection_statistics/models.py

```diff
@@ -25,11 +25,16 @@ class CountryWeeklyStatus(ConnectivityStatistics, TimeStampedModel, models.Model
     SCHOOL_MAPPED = 1
     STATIC_MAPPED = 2
     REALTIME_MAPPED = 3
+    DEFAULT = 4
```

Comment:
please rename to somethinng understandable. for example: country created. also we need to order statuses by flow, move new ones to the top